connector/atlassiancrowd/atlassiancrowd.go

```diff
@@ -80,16 +80,16 @@ type crowdAuthenticationError struct {
 }
 
 // Open returns a strategy for logging in through Atlassian Crowd
-func (c *Config) Open(_ string, logger log.Logger) (connector.Connector, error) {
+func (c *Config) Open(id string, logger *slog.Logger) (connector.Connector, error) {
 	if c.BaseURL == "" {
 		return nil, fmt.Errorf("crowd: no baseURL provided for crowd connector")
 	}
-	return &crowdConnector{Config: *c, logger: logger}, nil
+	return &crowdConnector{Config: *c, logger: logger.With("connector_type", "atlassiancrowd", "connector_id", id)}, nil
```

Comment:
Same thing about connectors: maybe use `slog.Group`?

Reply:
updated to use groups too